lib/system.g

```diff
@@ -67,11 +67,11 @@ BIND_GLOBAL( "GAPInfo", rec(
       ,
       rec( short:= "g", long := "gasinfo", default := 0,
            help := ["show GASMAN messages (full/all/no garbage","collections)", "(only available if GAP uses GASMAN)"] ),
-      rec( short:= "m", long := "minworkspace", default := "128m", arg := "<mem>",
+      rec( short:= "m", long := "initial-workspace", default := "128m", arg := "<mem>",
            help := ["set the initial workspace size"] ),
-      rec( short:= "o", long := "maxworkspace", default := "2g", arg := "<mem>",
+      rec( short:= "o", long := "warn-workspace", default := "2g", arg := "<mem>",
            help := [ "set workspace size where GAP will warn about", "excessive memory usage (GAP may allocate more)", "(available only if GAP uses GASMAN)"] ),
-      rec( short:= "K", long := "limitworkspace", default := "0", arg := "<mem>",
+      rec( short:= "K", long := "maximal-workspace", default := "0", arg := "<mem>",
```

Comment:
OK, thanks for fixing this.

Orthogonal to this PR: perhaps we should document those long names in the manual, too (see <https://www.gap-system.org/Manuals/doc/ref/chap3.html#X782751D5858A6EAF>) ? Perhaps we should file an issue to remind us about it... tomorrow...